context.go

```
@@ -38,7 +50,9 @@
 }
 
 func (c *Context) WithValue(key any, value any) {
-	ctx := context.WithValue(c.instance.UserContext(), key, value)
+	values := c.getGoravelContextValues()
+	values[key] = value
+	ctx := context.WithValue(c.instance.UserContext(), contextKey, values)
```

golangci-lint (1.62.2)

SA1029: should not use built-in type string as key for value; define your own type to avoid collisions

(staticcheck)